### DISCUSSION TITLE
Can I exclude unresolved log items from dnscrypt query log?

### DISCUSSION THREAD

**Discussion by retournerai** [DNSCrypt/dnscrypt-proxy#1943] [Q&A]
Is it possible to exclude all the 'unresolved' items from the query log DNScrypt can generate? 

The remote DNScrypt server blocks certain domains and fills my local DNScrypt client query log with these requests. Is it possible to exclude any unresolved items from the query log?

**Comment by jedisct1** [comment]
Hi!

This is not possible directly.

What can be done is log to a named pipe, and then use a command such as `grep --line-buffered -v <domain want to exlude>`

This is a feature that could be easily added if other people have a similar need.
